src/specBuilder/marks/markUtils.ts

```diff
@@ -338,6 +342,59 @@ export const getVoronoiPath = (children: MarkChildElement[], dataSource: string,
 	],
 });
 
+/**
+ * Gets the hover area for the mark
+ * @param children
+ * @param dataSource the name of the data source that will be used in the hover area calculation
+ * @param dimension the dimension for the x encoding
+ * @param metric the metric for the y encoding
+ * @param name the name of the component the hover area is associated with, i.e. `scatter0`
+ * @param scaleType the scale type for the x encoding
+ * @returns GroupMark
+ */
+export const getItemHoverArea = (
+	children: MarkChildElement[],
+	dataSource: string,
+	dimension: string,
+	metric: string,
+	name: string,
+	scaleType: ScaleType
+): GroupMark => {
+	return {
+		name: `${name}_hoverGroup`,
+		type: 'group',
+		marks: getHoverSizes().map((size, i) => ({
+			name: getHoverMarkName(name, i),
+			type: 'symbol',
+			from: { data: dataSource },
+			encode: {
+				enter: {
+					shape: { value: HOVER_SHAPE },
+					y: { scale: 'yLinear', field: metric },
+					fill: { value: 'transparent' },
+					stroke: { value: 'transparent' },
+					tooltip: getTooltip(children, name, false),
+					size: getHoverSizeSignal(size),
+				},
+				update: {
+					x: getXProductionRule(scaleType, dimension),
+				},
```

Comment:
```suggestion
				enter: {
					shape: { value: HOVER_SHAPE },
					y: { scale: 'yLinear', field: metric },
					fill: { value: 'transparent' },
					stroke: { value: 'transparent' },
					tooltip: getTooltip(children, name, false),
				},
				update: {
					x: getXProductionRule(scaleType, dimension),
					size: getHoverSizeSignal(size),
				},
```

Reply:
Looks like size is width dependent. If something is width dependent, it needs to be in the `update`. The reason for this is width can change dynamically. For example let's say someone resizes the window. This may adjust the chart width if it is set as a percentage. If `size` is not in the `update`, it won't be recalculated when this happens.